pkgs/development/libraries/zxing-cpp/default.nix

```diff
@@ -3,25 +3,36 @@
 , fetchFromGitHub
 , pkg-config
 , cmake
+, fmt
 }:
 
 stdenv.mkDerivation rec {
   pname = "zxing-cpp";
-  version = "1.1.1";
+  version = "1.2.0";
 
   src = fetchFromGitHub {
     owner = "nu-book";
     repo = pname;
     rev = "v${version}";
-    hash = "sha256-N2FTzsjxm3EE5Wqz7xt+FS4zQ60Ow4WbdX6Eo08ktek=";
+    hash = "sha256-M565VNKhSmYFmCMEI9UFuHWNZWeHrf9qzZkMAw9LUr4=";
   };
 
+  patches = [
+    # adapted from https://github.com/nu-book/zxing-cpp/commit/8db14eeead45e0f1961532f55061d5e4dd0f78be
```

Comment:
Hum, I'm not always happy with vendored patches on nixpkgs tree.

Can't you use `fetchpatch` to pick this patch?

What is the difference between the patch you are including here and the one you linked in this comment?

Reply:
Since dependencies are updated after the release of 1.2.0, the original commit introduces conflicts and cannot be applied as is.

Reply:
Turns out that these fetched deps are only used for building tests and we were not running tests anyway.